flake.nix

```diff
@@ -3,52 +3,57 @@
 
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-    flake-utils = { url = "github:numtide/flake-utils"; };
-    flake-compat = {
```

Comment:
You dropped flake-compat

Reply:
Oh no! Sorry... Thank you for noticing this!